# Generated for NaoQi method ALBehaviorManager.playDefaultProject
# Play default behaviors

---


